doc/classes/Camera.xml

```diff
@@ -76,25 +83,28 @@
 			<return type="float">
 			</return>
 			<description>
+				Returns the vertical (Y) offset of the Camera viewport.
 			</description>
 		</method>
 		<method name="get_zfar" qualifiers="const">
 			<return type="float">
 			</return>
 			<description>
+				Returns the far clip plane in world space units.
 			</description>
 		</method>
 		<method name="get_znear" qualifiers="const">
 			<return type="float">
 			</return>
 			<description>
+				Returns the near clip plane in world space units.
 			</description>
 		</method>
 		<method name="is_current" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
-				Return whether the Camera is the current one in the [Viewport], or plans to become current (if outside the scene tree).
+				Returns whether the Camera is the current one in the [Viewport], or plans to become current (if outside the scene tree).
```

Comment:
"Returns whether" should be changed to "Returns [code]true[/code]" as you did elsewhere.

Reply:
Fixed :)